packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -313,6 +313,7 @@ class Scrollable extends StatefulWidget {
   static Future<void> ensureVisible(
```

Comment:
The doc comment on this function should describe what `padding` does, and under what circumstances you might want to use it.